seacrowd/sea_datasets/xm3600/xm3600.py

```diff
@@ -0,0 +1,195 @@
+import jsonlines as jl
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{thapliyal-etal-2022-crossmodal,
+    title = "Crossmodal-3600: A Massively Multilingual Multimodal Evaluation Dataset",
+    author = "Thapliyal, Ashish V.  and
+      Pont Tuset, Jordi  and
+      Chen, Xi  and
+      Soricut, Radu",
+    editor = "Goldberg, Yoav  and
+      Kozareva, Zornitsa  and
+      Zhang, Yue",
+    booktitle = "Proceedings of the 2022 Conference on Empirical Methods in Natural Language Processing",
+    month = dec,
+    year = "2022",
+    address = "Abu Dhabi, United Arab Emirates",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.emnlp-main.45",
+    doi = "10.18653/v1/2022.emnlp-main.45",
+    pages = "715--729",
+}
+"""
+
+_DATASETNAME = "xm3600"
+
+_DESCRIPTION = """\
+Crossmodal-3600 dataset (XM3600 in short), a geographically-diverse set of 3600 images annotated with
+human-generated reference captions in 36 languages. The images were selected from across the world,
+covering regions where the languages are spoken, and annotated with captions that achieve consistency in
+terms of style across all languages, while avoiding annotation artifacts due to direct translation.
+The languages covered in the dataset include Filipino, Indonesian, Thai, and Vietnamnese
+"""
+
+_HOMEPAGE = "https://google.github.io/crossmodal-3600/"
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_URLS = {
+    "captions": "https://google.github.io/crossmodal-3600/web-data/captions.zip",
+    "images": "https://open-images-dataset.s3.amazonaws.com/crossmodal-3600/images.tgz",
+    "image_attributions": "https://google.github.io/crossmodal-3600/web-data/image_attributions.csv",
+}
+
+_SUPPORTED_TASKS = [Tasks.IMAGE_CAPTIONING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGS = ["fil", "id", "th", "vi"]
+
+
+class XM3600(datasets.GeneratorBasedBuilder):
+    """
+    Crossmodal-3600 dataset (XM3600 in short), a geographically-diverse set of 3600 images annotated with
+    human-generated reference captions in 36 languages. The images were selected from across the world,
+    covering regions where the languages are spoken, and annotated with captions that achieve consistency in
+    terms of style across all languages, while avoiding annotation artifacts due to direct translation.
+    The languages covered in the dataset include Filipino, Indonesian, Thai, and Vietnamnese
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [SEACrowdConfig(name=f"{_DATASETNAME}_{lang}_source", version=datasets.Version(_SOURCE_VERSION), description=f"{_DATASETNAME}_{lang} source schema", schema="source", subset_id=f"{_DATASETNAME}_{lang}",) for lang in _LANGS] + [
```

Comment:
Hmm, that's weird. When I run the Makefile I get something like this:

![image](https://github.com/SEACrowd/seacrowd-datahub/assets/12949683/9b57cd8d-de03-44c0-97bb-81f36b6625e6)

Aside from that, LGTM!

Reply:
I have no idea why, but when I ran that it changed my code that way. Anw, thank you for your time.

Reply:
I think it improves readability, esp for long one liner or statement within a statement (bcs you're defining a Config Class using list comprehension)

Reply:
Hi @IvanHalimP, would you consider to follow the formatter suggestions to improve code readability? Thanks